kubernetes/pybot/overlays/staging/ingress.yaml

```diff
@@ -0,0 +1,73 @@
+apiVersion: extensions/v1beta1
```

Comment:
We shouldn't need to make a new Ingress at all for Pyback.  Instead, let's roll it into the shared LBs for [Prod](https://github.com/OperationCode/operationcode_infra/blob/main/kubernetes/operationcode_python_backend/overlays/prod/ingress.yaml) and [Staging](https://github.com/OperationCode/operationcode_infra/blob/main/kubernetes/operationcode_python_backend/overlays/staging/ingress.yaml)

Reply:
I missed this in my last commit. Will correct in next commit.

Reply:
Actually, I get what you are saying but how to accomplish this is unclear

Reply:
So for both staging and prod, we'd add these lines directly to their ingress files:

to the [prod backend ingress rules](https://github.com/OperationCode/operationcode_infra/blob/main/kubernetes/operationcode_python_backend/overlays/prod/ingress.yaml):
```
  - host: pybot.k8s.operationcode.org
    http:
      paths:
      - path: /*
        backend:
          serviceName: pybot-service
          servicePort: 80
```

and the [staging backend ingress rules](https://github.com/OperationCode/operationcode_infra/blob/main/kubernetes/operationcode_python_backend/overlays/staging/ingress.yaml):
```
  - host: pybot-staging.k8s.operationcode.org
    http:
      paths:
      - path: /*
        backend:
          serviceName: pybot-service
          servicePort: 80
```